### ISSUE TITLE
Docs looks funny with extra space?

### ISSUE THREAD

**Issue by PaulWessel** [GenericMappingTools/gmt#5282]
**Description of the problem**

I just built the documentation and opened the front (gmt docs home) and there seems to be lots of extra space between the listing of the modules? Can anyone confirm this or is it an odd thing afflicting my laptop?

![bug](https://user-images.githubusercontent.com/26473567/120120284-f0aa8080-c137-11eb-91b7-67b8e1ab444c.png)

**Comment by PaulWessel** [comment]
Also get these two warnings if anyone knows what they mean:

```
/Users/pwessel/UH/RESEARCH/CVSPROJECTS/GMTdev/gmt-dev/rbuild/doc/rst/source/animations/anim08.rst:24: WARNING: Could not lex literal_block as "bash". Highlighting skipped.
/Users/pwessel/UH/RESEARCH/CVSPROJECTS/GMTdev/gmt-dev/rbuild/doc/rst/source/animations/anim13.rst:14: WARNING: Could not lex literal_block as "bash". Highlighting skipped.
```

**Comment by meghanrjones** [comment]
I get the same funny look when building the docs with macports sphinx. It looks fine when built in my pygmt development conda environment. I did a stepped install of the dependencies in the pygmt conda environment and found that the odd look went away after installing [myst-parser](https://myst-parser.readthedocs.io/en/latest/index.html). Not sure why this is now required for our docs.

I think the warnings relate to single quotes in the bash scripts, but I am not sure why the warnings occur for these two scripts but not others. If "EOF" and 'EOF' work the same in the scripts, we could update to double quotes to suppress the warnings.

**Comment by seisman** [comment]
Can you check your docutils version? The docutils v0.17 is known to break the style (https://github.com/GenericMappingTools/pygmt/issues/1176).

**Comment by PaulWessel** [comment]
py-docutils @0.17.1 (python, textproc)

**Comment by meghanrjones** [comment]
v0.17.1. Yes, docutils is the problem here. Installing myst-parser fixes the issues because it has docutils pinned at v0.15.2.

**Comment by PaulWessel** [comment]
Did you install myst-parser via port or conda, and if the former anything else that needs to be done?

**Comment by meghanrjones** [comment]
I installed with conda. I am not sure how to install older versions of ports, so here is an alternative conda solution:
```
conda create -n gmt-doc -c conda-forge sphinx docutils=0.16
conda activate gmt-doc
```

**Comment by meghanrjones** [comment]
MacPorts solution:
```
sudo port activate py38-docutils @0.16_1
sudo port select --set docutils py38-docutils
```

**Comment by PaulWessel** [comment]
I think that only works if you previously had 16.1 installed.  Unfortunately, I started form a blank slate on the M1 a few weeks ago when trying to fix that annoying link message.  So no 16_1 for me...

**Comment by meghanrjones** [comment]
Closed by #6012
